port-locking/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Terracotta, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.terracotta</groupId>
+    <artifactId>terracotta-utilities-parent</artifactId>
+    <version>0.0-SNAPSHOT</version>
```

Comment:
Are we using `0.0-SNAPSHOT` throughout this project? Shouldn't it be like `0.1-SNAPSHOT`?

Reply:
Yes. Why would it be 0.1?

Reply:
SNAPSHOT versions denote the product line, or the current development version. To me, `0.1-SNAPSHOT`, or `0.0.x-SNAPSHOT` would make sense.

Reply:
We use 2 digit snapshot versions in lots of the OSS stuff because they make more sense with the graph like nature of git - releases are tagged 'off' the main release and the branch version is left untouched. As for the '0' digit minor... we use that all the time with non-zero major version numbers... why not use it with a zero one too?

Reply:
Don't have many strong arguments, other than "it looks weird" :-D Go ahead please.